cmd/microcloud/add.go

```diff
@@ -17,6 +17,7 @@ type cmdAdd struct {
 
 	flagAutoSetup bool
 	flagWipe      bool
+	flagEncrypt   bool
```

Comment:
I don't think this flag is necessary. It would only be useful for `--auto` but the whole point of `--auto` is that we are deciding for you. So I think it would suffice to either enforce encryption or enforce no encryption in this case.

In the interactive setup we will ask you questions, and the preseed setup has its own key for encryption, so I think it would be safe to drop this.

Reply:
@masnax Then, for the --auto mode, I would suggest to NOT enforce encryption as this is a quite strong choice that has deep implication on the storage. But I think it also really depends on the philosophy of MicroCloud: what should be part of a default MicroCloud cluster.. are we putting the accent on simplicity or security ? (not to say that both aspects are always mutually exclusive, but in this case I think the question is relevant)

There is maybe also a third solution that is more complex: **encryption by default, when possible**. Which mean that we'll need some system validation logic. For example, is `dm_crypt` loaded in each cluster member containing the to be encrypted disks? If yes, proceed with encryption, else skip with an information message. What do you think?

Reply:
@mionaalex what do you think we should do here?